Gemfile

```diff
@@ -1,11 +1,14 @@
 ruby '>=2.3.1'
-source 'https://rubygems.org'
+source 'http://rubygems.org'
 
 # Middleman
-gem 'middleman', '~>4.2.1'
-gem 'middleman-syntax', '~> 3.0.0'
-gem 'middleman-autoprefixer', '~> 2.7.0'
-gem "middleman-sprockets", "~> 4.1.0"
-gem 'rouge', '~> 2.0.5'
-gem 'redcarpet', '~> 3.4.0'
-gem 'nokogiri', '~> 1.6.8'
+gem 'middleman', '>= 4.2.1'
+gem 'middleman-syntax', '>= 3.0.0'
+gem 'middleman-autoprefixer', '>= 2.7.0'
+gem "middleman-sprockets", ">= 4.1.0"
```

Comment:
why is your config different here? I don't think we should commit this

Reply:
I wasn't sure..so waited on your comment for this..removed it now